Lang/ChineseB5.pj.lang

```diff
@@ -1,426 +0,0 @@
-﻿/*********************************************************************************

```

Comment:
remember that if you remove this file, you have to remove line that call this file from: Installer.iss and package_zip.bat